apps/plea/views.py

```diff
@@ -92,11 +92,11 @@ def save(self, *args, **kwargs):
 
         return super(PleaOnlineForms, self).save(*args, **kwargs)
 
-    def render(self):
+    def render(self, request, request_context=dict):
```

Comment:
Is that default for `request_context` correct? It feels like it should be an empty `dict` rather than the dict type. And if that is the case probably safer to pass in `None` as the default and convert that to a `dict` in the method to avoid reuse of mutable arguments.